apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -13,15 +13,13 @@
  */
 function wpcom_should_limit_global_styles( $blog_id = 0 ) {
 	if ( ! $blog_id ) {
-		$blog_id = get_current_blog_id();
-	}
-
-	// Do not limit Global Styles on Atomic sites for now, because blog stickers are not exposed
-	// to these sites and the project is still in a development stage that requires sites to have
-	// a certain blog sticker before restricting the feature. This is a temporary check that will
-	// be removed as part of the public launch.
-	if ( ! defined( 'IS_WPCOM' ) || ! IS_WPCOM ) {
-		return false;
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+			$blog_id = get_current_blog_id();
+		} elseif ( defined( 'IS_ATOMIC' ) && IS_ATOMIC ) {
+			$blog_id = (int) Jetpack_Options::get_option( 'id' );
```

Comment:
Hmm... I haven't tested this yet, but I was looking at the code and how we determine if a site has a feature on the AT side. 
 
Here we assume that `get_current_blog_id` returns a different id than the id from Jetpack_Options. This blog_id is then called in[ wpcom_site_has_feature](https://github.com/Automattic/wp-calypso/blob/afd9e84bc9c5db89232f6dec98afadb062fddea6/apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php#L31) (non-zero blog_id). In this function, we call `wpcom_get_site_purchases` where we pass the blog_id. In there, we are checking if the $blog_id is different than get_current_blog_id(), if it's different, we return a WP_Error.

Reply:
Since these blog_ids should be different, wouldn't this throw an Error? 🤔

Reply:
Oh damn, you're right, nice catch! We have a bug in `wpcom_site_has_feature` we need to fix first. I'll open a PR to solve it

Reply:
https://github.com/Automattic/wpcomsh/pull/1196